src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/DecompressionHandler.cs

```diff
@@ -211,7 +211,7 @@ public DeflateDecompressedContent(HttpContent originalContent)
             { }
 
             protected override Stream GetDecompressedStream(Stream originalStream) =>
-                new DeflateStream(originalStream, CompressionMode.Decompress);
+                new ZLibStream(originalStream, CompressionMode.Decompress);
```

Comment:
Can we add a comment here about how, even though the HTTP compression scheme is called "deflate", what it really means is "zlib"? Or something like that. Otherwise it's kind of confusing why we aren't using DeflateStream here.

Reply:
Yup